src/index.html

```diff
@@ -4,10 +4,276 @@
+        </div>
+        <div class="gallery__item gallery__item--big">
+          <img
+          srcset="./images/gallery-image-earstick-small.jpg 320w, ./images/gallery-image-earstick-large.jpg 1260w"
+          src="./images/gallery-image-earstick-small.jpg"
+          sizes="(max-width: 1260px) 320px
+                1260px"
+          alt="Gallery photo earsticks"
+          class="gallery__image"
+          >
+        </div>
+        <p class="text--gallery">Audio</p>
+      </div>
+      <div class="gallery">
+        <div class="gallery__item gallery__item--big">
+          <img
+          srcset="./images/gallery-image-phone2-small.jpg 320w, ./images/gallery-image-phone2-large.jpg 1260w"
+          src="./images/gallery-image-phone2-small.jpg"
+          sizes="(max-width: 1260px) 320px
+                1260px"
+          alt="Gallery photo phone"
+          class="gallery__image"
+          >
+        </div>
+        <div class="gallery__item gallery__item--small">
+          <img
+          srcset="./images/gallery-image-charger-small.jpg 320w, ./images/gallery-image-charger-large.jpg 1260w"
+          sizes="(max-width: 1260px) 320px
+                1260px"
+          src="./images/gallery-image-charger-large.jpg"
+          alt="Gallery photo charger"
+          class="gallery__image"
+          >
+        </div>
+        <p class="text--gallery">Accessories</p>
+      </div>
+      <button class="button"><p class="text--button">View all</p></button>
+    </main>
+    <img
+      srcset="./images/footer-image-small.jpg 320w, ./images/footer-image-large.png 1260w"
+      src="./images/footer-image-small.jpg"
+      sizes="(max-width: 1260px) 320px
+          1260px"
+      alt="footer img unknown"
+      class="main__img"
+    >
+    <footer class="footer">
+      <div class="footer__container">
+        <h2 class="title title--secondary">About us</h2>
+        <p class="text">We’re a London-based tech company on a mission to remove barriers between people and technology.
+          <br>
+          <br>
+        To do this, we’re crafting intuitive, flawlessly connected products that improve our lives without getting in the way.
+        No confusing tech-speak. No silly product names. Just artistry, passion and trust. And products we’re proud to share with our friends and family. Simple.
+          <br>
+          <br>
+        Because beautiful tech shouldn’t be complicated.
+        </p>
+        <div class="contact footer__contact">
+          <h2 class="title contact__title title--secondary">Contact us</h2>
+          <form class="form contact__form">
+            <div class="form__item">
+              <label
+                for="name"
+                class="form__label"
+              >
+                Name
+              </label>
+              <input
+                id="name"
+                type="text"
+                placeholder="Name"
+                class="form__input"
+                required
+              >
```

Comment:
add name attribute for your inputs, check them to show error when empty

Reply:
Isn't 'required' attribute displaying an error when empty?